acceptance/setup/aio/pre-suite/021_InstallAristaModule.rb

```diff
@@ -0,0 +1,18 @@
+platforms = hosts.map{|val| val[:platform]}
+skip_test "No arista hosts present" unless platforms.any? { |val| /eos/ =~ val }
```

Comment:
I need to do more than work with the master in this script, hence the lack of a confine.